examples/spark2/src/main/java/org/apache/carbondata/examples/sdk/SDKS3Example.java

```diff
@@ -60,13 +63,19 @@ public static void main(String[] args) throws Exception {
         }
         writer.close();
         // Read data
+
+        EqualToExpression equalToExpression = new EqualToExpression(
```

Comment:
Please add examples for other expression supported. Like > , < ,in ,not in,, etc..

I think we need to review once the Expression classes can be given as API or we need to refactor to extract if it has internal details which SDK users not needed.

Reply:
ok, I add greater and less than , in, not in, NotEqual in carbonReaderTest

Yeah, we can optimize filter Api  in the future

Reply:
Yes, I am also worried about this exposure, I think it is better to create a simple DSL for user to pass the filter expression. For example: 
```
c1 > 3
c1 < 1 and c2 = 'apple'
c1 in (3,4,5)
c1 like ab*
```